packages/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -206,24 +208,28 @@ export const useTooltip = (
 
   if (state.triggerAriaAttribute === 'label') {
     // aria-label only works if the content is a string. Otherwise, need to use labelledby.
-    if (typeof state.content.children === 'string') {
-      triggerProps['aria-label'] = state.content.children as string;
+    if (typeof state.content === 'string') {
+      triggerProps['aria-label'] = state.content;
     } else {
       state.triggerAriaAttribute = 'labelledby';
     }
   }
 
   if (state.triggerAriaAttribute === 'labelledby' && !isServerSideRender) {
-    triggerProps['aria-labelledby'] = state.id;
+    triggerProps['aria-labelledby'] = state.root.id;
     // Always render the tooltip even if hidden, so that aria-labelledby refers to a valid element
     state.shouldRenderTooltip = true;
   } else if (state.triggerAriaAttribute === 'describedby' && !isServerSideRender) {
-    triggerProps['aria-describedby'] = state.id;
+    triggerProps['aria-describedby'] = state.root.id;
     state.shouldRenderTooltip = true;
   }
 
   // Apply the trigger props to the child, either by calling the render function, or cloning with the new props
-  state.children = applyTriggerPropsToChildren(state.children, triggerProps) as React.ReactElement;
+  if (typeof state.root.children === 'function') {
+    (state.root.children as React.ReactNode) = state.root.children(triggerProps);
+  } else if (state.root.children) {
+    (state.root.children as React.ReactNode) = React.cloneElement(onlyChild(state.root.children), triggerProps);
+  }
```

Comment:
I think this was a bad merge as the functionality on the right is abstracted in the function. I think this should be:

```suggestion
  state.root.children = applyTriggerPropsToChildren(state.root.children, triggerProps) as React.ReactElement;
```